lib/cacheman.ex

```diff
@@ -306,6 +312,9 @@ defmodule Cacheman do
   end
 
   def handle_call({:put_batch, key_value_pairs, put_opts}, _from, opts) do
+    # TODO temp logger
+    Logger.info("[Cacheman] OPTS: #{inspect(put_opts)}")
```

Comment:
How about using `Logger.debug/2` here?

Reply:
I planned to remove all the logs as soon as I figure out why this isn't performing as I expect :D But I will downgrading it to `debug` level instead and leave it here.